api/db/seeds/data/pole-emploi-sendings-builder.js

```diff
@@ -0,0 +1,52 @@
+const randomString = require('randomstring');
+const _ = require('lodash');
+const { POLE_EMPLOI_CAMPAIGN_ID } = require('./campaigns-builder');
+
+module.exports = function poleEmploisSendingsBuilder({ databaseBuilder }) {
+  const _generateFirstName = () => {
+    return randomString.generate({ length: 7, charset: 'alphabetic' });
+  };
+
+  const _generateLastName = () => {
+    return randomString.generate({ length: 10, charset: 'alphabetic' });
+  };
+
+  const _generateStatus = () => {
+    const possibleChoices = [{ isSuccessful: true, responseCode: '200' }, { isSuccessful: false, responseCode: '400' }];
+    return _.sample(possibleChoices);
+  };
+
+  const _generateDate = () => {
+    const mounth = Math.floor(Math.random() * 12) + 1;
+    const day = Math.floor(Math.random() * 29) + 1;
+    return new Date(`2020-${mounth}-${day}`);
+  };
+
+  _.times(300, async () => {
+    const user = await databaseBuilder.factory.buildUser({ firstName: _generateFirstName(), lastName: _generateLastName() });
```

Comment:
Pour éviter d'utiliser la fonction aléatoire (`randomstring`) et avoir des noms différents, tu peux juste faire :

```suggestion
  _.times(300, async (index) => {
    const user = await databaseBuilder.factory.buildUser({ firstName: `firstname-${index}`, lastName: `lastname-${index}` });
```

Reply:
j'y avais pas pensé et c'est mieux !!
Je change ça ^^